xml/ha_storage_protection.xml

```diff
@@ -685,7 +685,17 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
       to <literal>yes</literal> if <literal>msgwait</literal> is
       long, but your cluster nodes boot quickly.
       Setting this parameter to <literal>yes</literal> delays the start of
-      SBD on boot.  This is sometimes necessary with virtual machines.
+      SBD on boot. This is sometimes necessary with virtual machines.
+    </para>
+    <para>
+      The default delay length is the same as the <literal>msgwait</literal> timeout value.
+      Alternatively, you can specify an integer, in seconds, instead of <literal>yes</literal>.
+    </para>
+    <para>
+      If you enable <parameter>SBD_DELAY_START</parameter>, you must also check the SBD service file
+      to ensure that the value of <literal>TimeoutSec</literal> is greater than the value of
+      <parameter>SBD_DELAY_START</parameter>. For more information, see
+      <link xlink:href="https://www.suse.com/support/kb/doc/?id=000019356"/>.
```

Comment:
BTW, actually SBD_DELAY_START can be enabled/configured for the similar purpose in diskless SBD as well:

https://github.com/SUSE/doc-sleha/blob/main/xml/ha_storage_protection.xml#L932-L955

In that case, a setting SBD_DELAY_START=yes will take a delay value `SBD_WATCHDOG_TIMEOUT *2` rather than `msgwait`. Alternatively it can also be specified with an explicit time value.

Reply:
Thanks Yan, I've now added this!